### PR TITLE
Initial UWP Documentation.

### PR DESCRIPTION
**Pull request by wheatleyjj**
Added basic information, deployment process and project design pattern information

Closes https://github.com/langleyfoxall/technologies/issues/25.

### CODE REVIEW

File: platforms/desktop/windows.md [hunk 1]
```diff
@@ -1 +1,50 @@
...
+The Universal Windows Platform is our windows platform of choice, allowing you to develop and build once for Desktop, 
+Mobile, Xbox, Hololens and more.
+
+#### Things you should know
+
+- The UWP Platform is only compatible on Windows 10, with Windows 8 having a similar framework called Windows RT.
+- The UWP & WinRT platforms use .NET Core - as opposed to the full .NET Framework. For Windows 10 this has recently
+extended to .NET Core 2.0, which added a lot of extra API's.
+
+#### Design Patterns / Architecture
```
Comment (NilesB) [inline platforms/desktop/windows.md]: Might be worth mentioning views being built using XAML and which languages can be used (C++, C#, VB.NET).

Comment (wheatleyjj) [reply platforms/desktop/windows.md]: @NilesB Sorted - I've specified the preference as we have no active applications in C++ or VB.NET, and we certainly wouldn't start a new project in VB or C++, unless it was required for reasons I can't think of for LOB apps.

File: platforms/desktop/windows.md [hunk 2]
```diff
@@ -1 +1,54 @@
...
+
+UWP favours the MVVM design pattern (Model, View, View-Model), for separation of the views, business logic and models.
```
Comment (dextermb) [inline platforms/desktop/windows.md]: Might be worth linking the MVVM design pattern to a reference [such as a wiki page](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93viewmodel)

File: platforms/desktop/windows.md [hunk 3]
```diff
@@ -1 +1,54 @@
...
+
+This enables you to have more control over the structure and development of the projects, whilst speeding up the process
+tenfold.
+
+You can opt to use the full [MvvmLight Framework](https://www.nuget.org/packages/MvvmLight/) which offers more, but does
+inflict stronger opinions on your code, and less flexibility.
+
+#### Offline Databases
+
+When building on the UWP, we choose to use SQLite as our mobile database - this is because it closely matches the syntax
```
Comment (AlexCatch) [inline platforms/desktop/windows.md]: Maybe it's work checking out Realm, we use it for 99% of our mobile apps and it gives a nice abstraction on top of a much quicker DB with a beautiful ORM.

File: platforms/desktop/windows.md [hunk 4]
```diff
@@ -1 +1,54 @@
...
+ and structure of MSSQL databases - which is typically what you will be dealing with on the Microsoft stack.
+ 
+#### Deployment
+
+##### Continuous deployment
+
+Whilst in development, you will want to easily and consistently push new versions out to the customer for review.
+The easiest way of doing this is via an AdHoc service such as [HockeyApp](https://www.hockeyapp.net/). HockeyApp allows
```
Comment (AlexCatch) [inline platforms/desktop/windows.md]: This isn't really continuous deployment and should be moved to `Adhoc Deployment`

File: platforms/desktop/windows.md (hunk 3 above)
Comment (wheatleyjj) [reply platforms/desktop/windows.md]: Maybe - but I refuse to advocate anything on here that hasn't been proven commercially.

Comment (AlexCatch) [reply platforms/desktop/windows.md]: Exactly, just a comment to let you know of it's existence.

**Pull request merged**